Sources/Load.php

```diff
@@ -1942,6 +1943,21 @@ function loadTheme($id_theme = 0, $initialize = true)
 			'email' => '',
 			'ignoreusers' => array(),
 		);
+		// Note we should stuff $user_info with some guest values also...
+		$user_info['id'] = 0;
+		$user_info['is_guest'] = true;
+		$user_info['is_admin'] = false;
+		$user_info['is_mod'] = false;
+		$user_info['username'] = $txt['guest_title'];
+		$user_info['language'] = $language;
+		$user_info['email'] = '';
+		$user_info['smiley_set'] = '';
+		$user_info['permissions'] = array();
+		$user_info['groups'] = array();
+		$user_info['ignoreusers'] = array();
+		$user_info['possibly_robot'] = true;
+		$user_info['time_offset'] = 0;
+		$user_info['time_format'] = $modSettings['time_format'];
```

Comment:
Can't we just define all of these in a single array() statement like we do with `$context['user']` above?

Reply:
Yes, I'll need a day or two to change & retest....